test/TestUtilities/Test.Utility/JsonData.cs

```diff
@@ -4651,5 +4651,111 @@ public static class JsonData
         #endregion
 
 
+        #region PackageARegisteration
+
+        public static string PackageARegistration = @"{{
+  ""@id"": ""https://api.nuget.org/v3/registration0/PackageA/index.json"",
+  ""@type"": [
+    ""catalog:CatalogRoot"",
+    ""PackageRegistration"",
+    ""catalog:Permalink""
+  ],
+  ""commitId"": ""cb8b5792-c3e1-43ff-857f-9e3c1f92b0d1"",
+  ""commitTimeStamp"": ""2015-02-02T23:04:00.0649617Z"",
+  ""count"": 1,
+  ""items"": [
+    {{
+      ""@id"": ""https://api.nuget.org/v3/registration0/PackageA/index.json#page/0.5.0/1.0.0"",
+      ""@type"": ""catalog:CatalogPage"",
+      ""commitId"": ""cb8b5792-c3e1-43ff-857f-9e3c1f92b0d1"",
+      ""commitTimeStamp"": ""2015-02-02T23:04:00.0649617Z"",
+      ""count"": 1,
+      ""items"": [
+        {{
+          ""@id"": ""https://api.nuget.org/v3/registration0/PackageA/1.0.0.json"",
+          ""@type"": ""Package"",
+          ""commitId"": ""cb8b5792-c3e1-43ff-857f-9e3c1f92b0d1"",
+          ""commitTimeStamp"": ""2015-02-02T23:04:00.0649617Z"",
+          ""catalogEntry"": {{
+            ""@id"": ""https://api.nuget.org/v3/catalog0/data/2015.02.02.19.38.08/PackageA.1.0.0.json"",
+            ""@type"": ""PackageDetails"",
+            ""authors"": ""PackageA startup components contributors"",
+            ""description"": ""PackageA IAppBuilder startup interface"",
+            ""iconUrl"": """",
+            ""id"": ""PackageA"",
+            ""language"": """",
+            ""licenseUrl"": ""https://github.com/PackageA-contrib/PackageA-hosting/blob/master/LICENSE.txt"",
```

Comment:
Would server also return license expression in search metadata for newer clients? or we always going to read it from nuspec file explicitly to show in browse tab?

Reply:
Yes, the server will return a licenseExpression as per the design of @agr.